src/v2/coins/abstractUtxoCoin.js

```diff
@@ -1241,7 +1269,7 @@ class AbstractUtxoCoin extends BaseCoin {
 
       // By default, we will ignore P2WSH until we officially support it
       if (_.isUndefined(params.ignoreAddressTypes)) {
-        params.ignoreAddressTypes = [AbstractUtxoCoin.AddressTypes.P2WSH];
+        params.ignoreAddressTypes = [Codes.UnspentTypeTcomb('p2wsh')];
```

Comment:
Aren't we moving to officially support p2wsh now? :innocent:

Might want to put a fixme here to link the issue with which we'll enable p2wsh.

Reply:
Done